user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md

```diff
@@ -0,0 +1,139 @@
+---
+uid: Register_Catalog_Item
+---
+
+# Registering a Catalog item
+
+## What is a Catalog item?
+
+The content available in the Catalog module has always been provided by Skyline Communications. We have been using this module initially to share our connectors with all of our users. In a later stage, we have been adding app packages, Automation scripts and visual overviews. All of these items are called "Catalog items".
```

Comment:
I think we should be consistent if we use app packages or Application packages. Perhaps the first time we reference, we can add a link to the docs page of it.